### PR TITLE
timeout setting mixin

### PR DESCRIPTION
**Pull request by sirmmo**

### CODE REVIEW

**Comment by psychok7** [comment]
Could this PR be merged?? Currently we cannot set a timeout with `CacheResponseMixin`. This PR addresses that issue.

**Comment by auvipy** [comment]
@chibisov

File: rest_framework_extensions/cache/mixins.py [hunk 1]
```diff
@@ -8,20 +8,22 @@ class BaseCacheResponseMixin(object):
     # test_cache_reponse(view_instance, method, should_rebuild_after_method_evaluation)
     object_cache_key_func = extensions_api_settings.DEFAULT_OBJECT_CACHE_KEY_FUNC
     list_cache_key_func = extensions_api_settings.DEFAULT_LIST_CACHE_KEY_FUNC
+    object_cache_timeout = extensions_api_settings.DEFAULT_CACHE_RESPONSE_TIMEOUT
+    list_cache_timeout = extensions_api_settings.DEFAULT_CACHE_RESPONSE_TIMEOUT
 
 
 class ListCacheResponseMixin(BaseCacheResponseMixin):
-    @cache_response(key_func='list_cache_key_func')
+    @cache_response(key_func='list_cache_key_func', timeout='list_cache_timeout')
```
Comment (psychok7) [inline rest_framework_extensions/cache/mixins.py]: i am getting a `ValueError: invalid literal for int() with base 10: 'list_cache_timeout'`.
To fix it i used `@cache_response(key_func='list_cache_key_func', timeout=list_cache_timeout)` instead.

**Comment by SerhiyRomanov** [comment]
Any updates? I've emailed to @chibisov, and he said that there are maintainers who can merge it. 
@auvipy ?

**Comment by auvipy** [comment]
there is some issue in this pr which needs to be addressed

**Comment by SerhiyRomanov** [comment]
@auvipy, i've just created new https://github.com/chibisov/drf-extensions/pull/244 which resolve this issues

**Pull request closed without merging**